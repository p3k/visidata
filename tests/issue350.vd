sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/issue350.json	o	
issue350	season		addcol-expr	episode	=	
issue350	episode		rename-col	episode	^	
issue350	1		slide-right		L	slide current column right
issue350	2		cache-col		z'	add/reset cache for current column
issue350	2		type-int		#	set type of current column to int
