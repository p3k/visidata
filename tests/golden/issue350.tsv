season	episode	episode	rating	votes
1	0	0	7	1415
1	1	1	8	3441
1	2	2	8	2973
1	3	3	7	2994
